Dockerfile.dev

```diff
@@ -1,7 +1,10 @@
-FROM python:3.7
+FROM python:3.8
 
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
+RUN \
+    && apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
&& should be removed from the beginning of this line -> can't build container

Reply:
I'm also running into both these issues.
It's unclear to me why the ppa was added, since none of the installed packages live in the ppa.

Reply:
I opened #34747with fixes.

Reply:
thanks =)
yeah I wasn't sure why it got added there. Maybe it was an purpose. Thats why I've asked to fix it instead of creating a PR with removing it^^